src/transformers/models/hubert/modeling_tf_hubert.py

```diff
@@ -416,11 +416,6 @@ def _normalize_kernel(self):
 
     def build(self, input_shape):
         if not self.built:
-            input_shape = input_shape.as_list()
-            # If a specific input shape is passed in, we need to modify it to account for padding
-            # Not necessary if those portions of the shape are None
-            if input_shape[-2] is not None:
-                input_shape[-2] += self.explicit_padding * 2
             super().build(input_shape)
```

Comment:
Do we still want this super().build() call here?

Reply:
Probably not, but it should be harmless - I don't even think the `super()` method really does anything in TF anymore!